ui/app/App.js

```diff
@@ -605,7 +605,7 @@ class App extends React.Component {
 
     this.Notebook.run = async (path, options) => {
       const { LocalFilesystemHandles = [] } = this.state;
-      for (const { handle } of LocalFilesystemHandles) {
+      for (const [, handle] of LocalFilesystemHandles) {
```

Comment:
What does the , here do?

Reply:
([a, b] = [1, 2]) assigns 1 to a and 2 to b.

([, b] = [1, 2]) ignores 1 and assigns 2 to b.